os_stub/spdm_device_secret_lib_sample/raw_data_key.h

```diff
@@ -0,0 +1,438 @@
+/**

```

Comment:
This file includes data structure definition. It should be c file not h file.

Reply:
Thanks. I have made the script generate `.c` file automatically.